samples/core/retry/retry_test.py

```diff
@@ -13,11 +13,10 @@
 # limitations under the License.
 
 import kfp
```

Comment:
feel free to consider this out of scope, but there's a couple of instances in the updated files, where the direct kfp imports don't look like they are used at all, seems like we can just drop them all together